adapters/quick/src/main/scala/caliban/GraphiQLHandler.scala

```diff
@@ -11,9 +11,30 @@ object GraphiQLHandler {
    * Creates a handler which serves the GraphiQL UI from CDN.
    *
    * @param apiPath The path at which the API can be introspected.
+   * @param wsPath The path at which the WS subscription can be introspected.
    *
    * @see [[https://github.com/graphql/graphiql/tree/main/examples/graphiql-cdn]]
    */
+  def handler(apiPath: String, wsPath: Option[String]): RequestHandler[Any, Nothing] = {
+    val headers = Headers(Header.ContentType(MediaType.text.html).untyped)
+    zio.http.handler { (req: Request) =>
+      val body = wsPath.fold(html(apiPath, req.path.encode))(html(apiPath, req.path.encode, _))
+      Response(
+        Status.Ok,
+        headers,
+        Body.fromString(body)
+      )
+    }
+  }
+
+  /**
+   * Creates a handler which serves the GraphiQL UI from CDN.
+   *
+   * @param apiPath The path at which the API can be introspected.
+   *
+   * @see [[https://github.com/graphql/graphiql/tree/main/examples/graphiql-cdn]]
+   */
+  @deprecated("Use overloaded method also providing optional graphql subscription param", since = "2.10.1")
```

Comment:
To reduce code duplication, how about making this one call the other one with `wsPath = None`?

Reply:
you are right! fixed